index.html

```diff
@@ -1,15 +1,141 @@
 <!DOCTYPE html>
-<html >
+<html lang="en-US">
 	<head>
 		<meta charset="UTF-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
-		<title></title>
-		<style >
-			
-		</style>
+		<title>Intuit. Signup</title>
 		<link rel="stylesheet" href="stylesheet.css" />
 	</head>
 	<body>
-		<h1>Intuit Login Page clone</h1>
+		<section>
+			<nav id="navbar">
+				<div class="link_container">
+					<div class="left_nav">
+						<div>Already have an Intuit account?</div>
+						<a href="#" class="link">I forgot my User ID or Password</a>
+					</div>
+					<div class="right_nav">
+						<button id="signin_button">
+						<span class="lock"></span>
+						Sign Up
+						</button>
+					</div>
+				</div>
+			</nav>
+			<header class="header">
+				<div class="title">
+					<a href="#"></a>
+				</div>
+			</header>
+			<section>
+				<div>
+					<header class="head">
+						<div>
+							<ul>
+								<li class="list_item list_icon" id="mint_icon"></li>
+								<li class="list_item list_text" id="mint_text"></li>
+								<li class="list_item list_icon" id="qb_icon"></li>
+								<li class="list_item list_text" id="qb_text"></li>
+								<li class="list_item list_icon" id="cb_icon"></li>
+								<li class="list_item list_text" id="cb_text"></li>
+							</ul>
+						</div>
+					</header>
+				</div>
+				<div class="main">
+					<section >
+						<header class="form_header">
+							<span class="form_title">Create an Intuit account</span>
+							<span class="form_title_des">
+								<span class="div">One account for everything Intuit, including Mint.
+									<a href="#" class="link">Learn more</a>
+								</span>
+							</span>
+						</header>
+						<form>
+							<fieldset>
+								<label class="label">Email address</label>
+								<div class="form_div">
+									<input type="email" name="email" class="form_input" />
+								</div>
```

Comment:
:bulb: Consider associating the `<label>` with the `<input>` element, to do this you need to give the `<input>` an `id` attribute. The `<label>` then needs a `for` attribute whose value is the same as the input's id. 
- Example 1: 

``` html
  <label for="email">Email address</label> <br>
  <input type="email" name="email" id="email">
```
- Example 2: Alternatively, you can nest the `<input>` directly inside the `<label>` like this: 

``` html
  <label for="email">Email address <br>
       <input type="email" name="email" id="email">
  </label> 
```

> #### Associating a `<label>` with an `<input>` element offers some major advantages:
> 
> - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
> 
> - You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

You can learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)


Please do this for other input elements.

Reply:
Thanks for the review. I'll work on this.